views/frontend/post.php

```diff
@@ -32,68 +32,36 @@
                             </blockquote>
                         <?php
                     }
-                }else echo "Aucun commentaire n'a été publié... Soyez le premier!";
-            ?>
-
-            <h4>Commenter:</h4>
-
-            <?php
-                if(isset($_POST['submit'])){
-                    $name = htmlspecialchars(trim($_POST['name']));
-                    $email = htmlspecialchars(trim($_POST['email']));
-                    $comment = htmlspecialchars(trim($_POST['comment']));
-                    $errors = [];
-
-                    if(empty($name) || empty($email) || empty($comment)){
-                        $errors['empty'] = "Tous les champs n'ont pas été remplis";
-                    }else{
-                        if(!filter_var($email, FILTER_VALIDATE_EMAIL)){
-                            $errors['email'] = "L'adresse email n'est pas valide";
-                        }
-                    }
-
-
-                    if(!empty($errors)){
-                        ?>
-                            <div class="card red">
-                                <div class="card-content white-text">
-                                    <?php
-                                        foreach($errors as $error){
-                                            echo $error."<br/>";
-                                        }
-                                    ?>
-                                </div>
-                            </div>
-                        <?php
-                    }else{
-                        $model_comment->insert_comment($name,$email,$comment);
-                        ?>
-                            <script>
-                                window.location.replace("index.php?page=post&id=<?= $_GET['id'] ?>");
-                            </script>
-                        <?php
-                    }
+                }else{ 
+                    echo "Aucun commentaire n'a été publié... Soyez le premier!";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5349276)